lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -436,11 +436,11 @@ def process_device(device_info)
     end
 
     def process_optional_hck_network
-      return unless @config['transfer_net_enabled']
+      return unless @options['share_on_host_path']
 
       dev = @nm.transfer_device_command(@config['transfer_net_device'],
                                         @config['share_on_host_net'],
-                                        @config['share_on_host_path'],
+                                        @options['share_on_host_path'],
```

Comment:
It's better to make it configurable with `@config` so that I can leave file sharing always enabled on my development environment, for example.

Reply:
What about your comment 2 week ago:
```
It is awkward to create a transfer network when transfer_net_enabled is false. What about removing it?
```

Reply:
I'm not for keeping `transfer_net_enabled` in `@config`, but I'm suggesting keeping `share_on_host_path` in `@config`.

Reply:
What I'm suggesting is:
```Ruby
      path = @options['share_on_host_path'] || @config['share_on_host_path']
      return unless path

      dev = @nm.transfer_device_command(@config['transfer_net_device'],
                                        @config['share_on_host_net'],
                                        path,
                                        full_replacement_list)
      @device_commands << dev
```